cmd/metal-api/internal/datastore/image.go

```diff
@@ -35,3 +47,210 @@ func (rs *RethinkStore) DeleteImage(i *metal.Image) error {
 func (rs *RethinkStore) UpdateImage(oldImage *metal.Image, newImage *metal.Image) error {
 	return rs.updateEntity(rs.imageTable(), newImage, oldImage)
 }
+
+// MigrateMachineImages check images of all machine allocations and migrite them to semver images
+// must be executed only once.
+func (rs *RethinkStore) MigrateMachineImages(machines metal.Machines) (metal.Machines, error) {
```

Comment:
Why did you make a machines parameter and not just always get all the machines? Is there a specific reason?

Reply:
this helps during tests